backend/djangoindia/db/models/user.py

```diff
@@ -113,6 +114,12 @@ def __str__(self):
 
     def save(self, *args, **kwargs):
         self.email = self.email.lower().strip()
+        try:
+            # Trying to find the hash used on the password, if it couldn't find a one, exception will be thrown, ie, password isn't hashed :-)
+            identify_hasher(self.password)
+        except:
```

Comment:
It's not ideal to use bare except. You can use `except Exception:` if you're not sure what errors will be thrown.